src/logo.tsx

```diff
@@ -0,0 +1,352 @@
+                    fill="#00B6F1"
+                />
+                <path d="M45.2905 8.75794V18.8042H48.8477V46.5732H61.8442V8.75794H45.2905Z" fill="#00B6F1" />
+                <path
+                    d="M99.2576 46.5732H87.0676L86.7996 43.7674C84.8108 46.1975 81.8553 47.0592 79.0705 47.117C60.5884 47.3325 60.5884 16.6525 79.0705 16.8627C81.8553 16.8627 84.8108 17.727 86.7996 20.1572L87.0676 17.3435H99.2576V46.5732ZM86.3188 31.9846C86.3188 26.3126 78.3191 26.3704 78.3191 31.9846C78.3191 37.6067 86.3188 37.6619 86.3188 31.9846Z"
+                    fill="#00B6F1"
+                />
+                <path
+                    d="M134.858 17.3435H121.972V33.1222C121.972 35.2844 120.359 36.309 118.748 36.309C117.13 36.309 115.843 35.2345 115.843 33.1774V17.3435H102.949V33.2378C102.949 41.072 106.009 46.9043 114.661 47.1171C117.506 47.1696 120.467 46.2501 122.5 43.6571L122.773 46.5733L134.861 46.5917L134.858 17.3435Z"
+                    fill="#00B6F1"
+                />
+            </svg>
+        );
+    }
+
+    return (
+        <svg height={size} viewBox="0 0 72 72" fill={color}>
+            <path d="M55.8 52.2003H18V63.0003H55.8V52.2003Z" fill="#7814B2" />
+            <path
+                d="M38.3805 36.431H34.6798V31.658H38.3805C40.3136 31.658 41.0868 32.865 41.0868 34.0171C41.0868 35.2789 40.3136 36.431 38.3805 36.431ZM34.7351 19.0948H37.4966C40.2029 19.0948 40.0373 23.0447 37.4966 23.0447H34.7351V19.0948ZM49.1503 26.2267C52.1327 25.1844 53.7342 22.3864 53.7895 19.369C53.9552 13.7182 49.4819 9 43.8483 9H18V19.0398H21.6454V36.7602H18V46.8H43.7377C50.3654 46.8 55.5022 42.3013 55.7784 35.8276C56.0545 30.89 53.6792 27.9275 49.1503 26.2267Z"
+                fill="#00B5F2"
+            />
+        </svg>
+    );
+};
+
+type LogoBaseProps = {
+    skinName: SkinName;
+    size?: number;
+    logoType?: LogoType;
+};
+
+const LogoBase: React.FC<LogoBaseProps> = ({size = 48, skinName, logoType = 'isotipo'}) => {
+    switch (skinName) {
+        case 'Movistar':
+            return <MovistarLogoImage size={size} logoType={logoType} />;
+        case 'Vivo':
+            return <VivoLogoImage size={size} logoType={logoType} />;
+        case 'O2':
+        case 'O2-classic':
+            return <O2LogoImage size={size} logoType={logoType} />;
+        case 'Telefonica':
+            return <TelefonicaLogoImage size={size} logoType={logoType} />;
+        case 'Blau':
+            return <BlauLogoImage size={size} logoType={logoType} />;
+        // todo WEB-761 what to do for unknown skinnames?
```

Comment:
just for the repetition sake: we've said on teams this is OK by now, but let's confirm it here

Reply:
🤔 it's not such a big problem, it's technically quick and easy, I'd say it's simply a matter of choosing the "missing" logo. I'll open a ticket for it, so that you can think about the choice properly!